lua/core/mappings.lua

```diff
@@ -27,6 +27,31 @@ M.misc = function()
 
       -- use ESC to turn off search highlighting
       map("n", "<Esc>", ":noh <CR>")
+       ---- jump and align in the middle
```

Comment:
Can you use https://nvchad.netlify.app/contribute#how-to-remove-or-edit-commits-from-your-pr

To remove the `fix typos commit`, using `squash` please! :)